lib/bridge/IrcBridge.js

```diff
@@ -303,6 +303,15 @@ IrcBridge.prototype.run = Promise.coroutine(function*(port) {
     this._startedUp = true;
 });
 
+IrcBridge.prototype.kill = function() {
+    if (!this._bridge.getBot()) {
+        log.info("No bot client to kill");
+        return Promise.resolve({});
```

Comment:
This seems weird. Why should there being no bot affect the `ClientPool`? Why aren't we just directly calling `killAllClients()`?